src/ConfidentialLedger/examples/New-AzConfidentialLedger.md

```diff
@@ -0,0 +1,25 @@
+### Example 1: Create a new Confidential Ledger
+```powershell
+PS C:\> New-AzConfidentialLedger `
+  -Name test-ledger `
+  -ResourceGroupName rg-000 `
+  -SubscriptionId 00000000-0000-0000-0000-000000000000 `
+  -AadBasedSecurityPrincipal `
+      @{
+          LedgerRoleName="Administrator"; 
+          PrincipalId="34621747-6fc8-4771-a2eb-72f31c461f2e"; 
+          TenantId="bce123b9-2b7b-4975-8360-5ca0b9b1cd08"
+      } `
+  -CertBasedSecurityPrincipal `
+      @{
+          Cert="-----BEGIN CERTIFICATE-----MIIBsjCCATigAwIBAgIUZWIbyG79TniQLd2UxJuU74tqrKcwCgYIKoZIzj0EAwMwEDEOMAwGA1UEAwwFdXNlcjAwHhcNMjEwMzE2MTgwNjExWhcNMjIwMzE2MTgwNjExWjAQMQ4wDAYDVQQDDAV1c2VyMDB2MBAGByqGSM49AgEGBSuBBAAiA2IABBiWSo/j8EFit7aUMm5lF+lUmCu+IgfnpFD+7QMgLKtxRJ3aGSqgS/GpqcYVGddnODtSarNE/HyGKUFUolLPQ5ybHcouUk0kyfA7XMeSoUA4lBz63Wha8wmXo+NdBRo39qNTMFEwHQYDVR0OBBYEFPtuhrwgGjDFHeUUT4nGsXaZn69KMB8GA1UdIwQYMBaAFPtuhrwgGjDFHeUUT4nGsXaZn69KMA8GA1UdEwEB/wQFMAMBAf8wCgYIKoZIzj0EAwMDaAAwZQIxAOnozm2CyqRwSSQLls5r+mUHRGRyXHXwYtM4Dcst/VEZdmS9fqvHRCHbjUlO/+HNfgIwMWZ4FmsjD3wnPxONOm9YdVn/PRD7SsPRPbOjwBiE4EBGaHDsLjYAGDSGi7NJnSkA-----END CERTIFICATE-----"; 
```

Comment:
CERTIFICATE should be hidden otherwise we aren't able to pass security check.

Reply:
This is a public key and we include it in the example for all of our other public releases too (Swagger, Azure CLI, etc). Will your CI specifically reject this, or is it ok to keep it here?

Reply:
It's risky. I would like to suggest you to represent the key with a obscure asterisk， like 
```
          Cert="-----BEGIN CERTIFICATE-----**************************-----END CERTIFICATE-----"; 
```

Reply:
I've hidden it in the examples and docs. However, it is still part of the tests and recordings, and it has to be there because we validate the certificate in the resource provider.

Just wanted to reiterate that this certificate is intended to be public information and has no security value.

Reply:
Got it. I just want to hide it from docs to avoid security risk alert.